Bundle/FormBundle/Resources/public/js/slug.js

```diff
@@ -22,7 +22,7 @@ function displaySlugIcons(inputSlug, slug, allowEmpty){
         }
     }else{
         if (allowEmpty && slug != 'undefined') {
-            correctSlugIcon.removeClass('v-hidden');
+            correctSlugIcon.addClass('v-hidden');
```

Comment:
if slug is empty but allowed, why you you hide correctSlugIcon ?

Reply:
It's pure ux, the green tick should be used only to validate that the entered input is a well formed slug. When there isn't any value, this is also valid (if allowEmpty) but like any other form field and there isn't a green check for each other field.